markdown/extensions/footnotes.py

```diff
@@ -49,7 +49,12 @@ def __init__(self, *args, **kwargs):
             "BACKLINK_TEXT":
                 ["&#8617;",
                  "The text string that links from the footnote "
-                 "to the reader's place."]
+                 "to the reader's place."],
+            "BACKLINK_TITLE":
+                ["Jump back to footnote %d in the text",
+                 "The text string used for the title HTML attribute"
+                 "of the backlink. %d will be replaced by the"
+                 "footnote number."]
```

Comment:
By using this way of string concatenation, you lose spaces:
```python
>>> footnotes.FootnoteExtension().getConfigInfo()[3]
('BACKLINK_TITLE', 'The text string used for the title HTML attributeof the backlink. %d will be replaced by thefootnote number.')
```

Please add spaces after ‘attribute’ and ‘the’ at the ends of lines.